ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -216,6 +217,25 @@ def needs_review(self):
             return self.Status.UNDECIDED
 
 
+class RevertedEditByAnotherUser(BaseReviewRequiredDecider):
+    def review_description_text(self):
+        return "Edit reverted by another user"
+
+    def needs_review(self):
+        from candidates.models import LoggedAction
+
+        if self.logged_action.action_type == "person-revert":
+            recent_revert_qs = LoggedAction.objects.filter(
+                person=self.logged_action.person,
+                action_type="person-revert",
```

Comment:
Can you convert this to use `ActionType.PERSON_REVERT`? As a rule, we should use that rather than the value